L1Trigger/TrackFindingTracklet/src/VMRouterTable.cc

```diff
@@ -98,6 +99,40 @@ void VMRouterTable::init(unsigned int layerdisk) {
       }
     }
   }
+
+  if (settings_.writeTable()) {
+    if (not std::filesystem::exists(settings_.tablePath())) {
+      system((string("mkdir -p ") + settings_.tablePath()).c_str());
```

Comment:
check if successful

Reply:
DONE
--  P.S. As this file/directory creation code appears in dozens of places, a future PR (almost ready) will move it to a single common function, to reduce duplicate code.